app/handlers/queue_worker.py

```
@@ -66,10 +69,27 @@
+                    
+                    job_tracker = self.job_tracker_manager.create_tracker(
+                        worker_id=self.worker_id,
+                        queue_name=queue_name
+                    )
+                    
+                    not_claimed = await job_tracker.try_claim(message_id=job.get("id"))
+                    
+                    if not not_claimed:
+                        do_job = False
+                    else:
+                        do_job = True
```

Ruff (0.12.2)

Use ternary operator `do_job = False if not not_claimed else True` instead of `if`-`else`-block

Replace `if`-`else`-block with `do_job = False if not not_claimed else True`

(SIM108)